### PR TITLE
EVG-5379 populate manifest when a version is created

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

File: scripts/indexes.js [hunk 1]
```diff
@@ -153,3 +153,6 @@ db.daily_test_stats.createIndex({ "_id.date": 1 }, { expireAfterSeconds: 26 * 7
 
 //======daily_task_stats======//
 db.daily_task_stats.createIndex({ "_id.date": 1 }, { expireAfterSeconds: 26 * 7 * 24 * 3600 })  // 26 weeks TTL
+
+//======manifest======//
+db.daily_task_stats.createIndex({ "project": 1, "revision": 1 })
```
Comment (bsamek) [inline scripts/indexes.js]: Remember a ticket for this if there isn't one.

File: service/api_plugin_manifest.go [hunk 2]
```diff
@@ -40,7 +38,12 @@ func (as *APIServer) manifestLoadHandler(w http.ResponseWriter, r *http.Request)
 	}
 
 	// try to get the manifest
-	currentManifest, err := manifest.FindOne(manifest.ById(task.Version))
+	v, err := version.FindOne(version.ById(task.Version))
```
Comment (bsamek) [inline service/api_plugin_manifest.go]: nil check for `v`

File: scripts/indexes.js (hunk 1 above)
Comment (john-m-liu) [reply scripts/indexes.js]: made a ticket

File: service/api_plugin_manifest.go (hunk 2 above)
Comment (john-m-liu) [reply service/api_plugin_manifest.go]: added

File: repotracker/repotracker.go [hunk 3]
```diff
@@ -454,6 +466,46 @@ func AddBuildBreakSubscriptions(v *version.Version, projectRef *model.ProjectRef
...
+		if err != nil {
+			return nil, errors.Wrapf(err, "problem retrieving getting git branch for module %s", module.Name)
+		}
+
+		modules[module.Name] = &manifest.Module{
+			Branch:   module.Branch,
+			Revision: *gitBranch.Commit.SHA,
+			Repo:     repo,
+			Owner:    owner,
+			URL:      *gitBranch.Commit.URL,
```
[the remark this answers is not in the document]
Comment (john-m-liu) [reply repotracker/repotracker.go]: added some nil checking

**Pull request merged**